docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -889,15 +890,38 @@ testcafe chrome my-tests --sf
 
 ### --ts-config-path \<path\>
 
-Enables TestCafe to use a custom [TypeScript configuration file](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options) and specifies its location.
+Deprecated as of [TestCafe v.X.X.X] in favour of the [`--compiler-options`](#--compiler-options-options) parameter.
+
+*Related configuration file property*: [tsConfigPath](configuration-file.md#tsconfigpath).
+
+### --compiler-options \<options\>
+
+Specifies test compilation settings. The current version of TestCafe can only configure the [TypeScript compiler](../../../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options).
 
 ```sh
-testcafe chrome my-tests --ts-config-path /Users/s.johnson/testcafe/tsconfig.json
+testcafe chrome my-tests --compiler-options typescript.options.lib=ES5,WebWorker;typescript.typesRoot=’this value contains spaces'
 ```
 
-You can specify an absolute or relative path. Relative paths are resolved against the current directory (the directory from which you run TestCafe).
+The `--compiler-options` parameter accepts the compiler options listed in the [official TypeScript documentation](https://www.typescriptlang.org/docs/handbook/compiler-options.html), as well as two additional options: `typescript.configPath` and `typescript.customCompilerModulePath`.
 
-*Related configuration file property*: [tsConfigPath](configuration-file.md#tsconfigpath).
+Specify the `typescript.configPath` compiler option to load an external TypeScript configuration file:
```

Comment:
We don't have an `internal` configuration file. 

`to load an external TypeScript configuration file` - is not clear why it's necessary for customer.

Consider such variant:
```js
Specify the `typescript.configPath` compiler option to load TypeScript compiler settings from the target `.tsconfig` file
```